tests/testing_harness.py

```diff
@@ -340,8 +340,9 @@ def _build_inputs(self):
 
     def _get_inputs(self):
         """Return a hash digest of the input XML files."""
-        xmls = ['geometry.xml', 'materials.xml', 'settings.xml',
-                'tallies.xml', 'plots.xml']
+        #xmls = ['geometry.xml', 'materials.xml', 'settings.xml',
+        #        'tallies.xml', 'plots.xml']
+        xmls = ['model.xml']
```

Comment:
This is causing a bunch of tests to fail because it ends up writing nothing for inputs_true.dat. You'll need to fix this to get tests passing.